test/system_category_test.cpp

```diff
@@ -110,14 +111,7 @@ int main()
 
     // message
 
-    for( int i = -2; i < 1024; ++i )
-    {
-        test_message( cat, i );
-    }
-
-    test_message( cat, 5810 );
-
-    for( int i = 10000; i < 11032; ++i )
+    for( int i = -2; i <= 15999; ++i )
```

Comment:
What's the purpose of this and the preceding change?

Reply:
To cover the bug on more locales. Here https://www.boost.org/development/tests/develop/developer/output/igaztanaga-develop-gcc-5-4c++03-boost-bin-v2-libs-system-test-system_category_test-test-gcc-5-4c+-dbg-dbg-symbl-off-thrd-mlt-vsblt-hdn.html the bug is triggered with precious range, with the change I it catches on my machine too.